src/ChartInternal/internals/format.ts

```diff
@@ -46,6 +46,11 @@ export default {
 	},
 
 	defaultValueFormat(v): number|string {
+		// TODO We could extract this if block
+		// as defaultRangeValueFormat like thing.
+		if (isRange(v)) {
+			return `${Math.min(...v)} ~ ${Math.max(...v)}`;
```

Comment:
The data format for ranged bar data, should be following as `[start, end]`. There's no need to get min/max value. To get reason for this, it should be able treat data values as `[end, start]` case, which isn't proper data format that will be accepted.

```js
// is enough return as:
return v.join(" ~ ")
```

Reply:
Okay I got it, so you mean the following code should not render any thing right?

```js
{
  data: {
    columns: [
      ["data1",  [200, 100], [300, 200]]
    ],
    type: "bar"
  }
}
```

And I have another question, the above code should throw an error or just does't render quietly?

Reply:
Okay I'll make it throw an error :)